calculator.html

```diff
@@ -0,0 +1,26 @@
+2.- do a ver simple calculator interface that works for (+ * /)
+
+<html>
+<head>
+<title>HTML Calculator</title>
```

Comment:
inside head is where you insert both the css and the html.
<script src="nameOfFile.js"></script>
<link rel="old stylesheet" href="nameofFile.css">